indico/modules/categories/models/categories.py

```diff
@@ -256,6 +256,21 @@ def can_create_events(self, user):
         """Check whether the user can create events in the category."""
         return user and (not self.event_creation_restricted or self.can_manage(user, role='create'))
 
+    def move(self, target):
+        """Move the category into another category."""
+        assert not self.is_root
```

Comment:
Do we check a category not being able to be moved to any of the subcategories? Shouldn't this check belong there or the other here?